subprojects/enterprise/src/main/java/org/gradle/internal/enterprise/GradleEnterprisePluginService.java

```diff
@@ -32,6 +32,13 @@ public interface GradleEnterprisePluginService {
      */
     BuildOperationNotificationListener getBuildOperationNotificationListener();
 
+    /**
+     * Notified when the execution phase starts.
+     */
+    default GradleEnterprisePluginExecutionPhaseListener getExecutionPhaseListener() {
+        return NoopMainBuildExecutionListener.INSTANCE;
```

Comment:
this is to keep the compatibility with older versions of the build scan plugin ?

Reply:
Yes.